### PR TITLE
[spi_device] Waiving CDC violations through prim_flop_2sync

### PR DESCRIPTION
**Pull request by eunchan**
Adding W_CNTL waivers that are passing through prim_flop_2sync.

### CODE REVIEW

File: hw/top_earlgrey/cdc/cdc_waivers.spi_device.tcl [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#  ControlSignal==""
+#  ReconSignal==""
+#  MultiClockDomains=="IO_DIV2_CLK::IO_DIV4_CLK"
+
+set_rule_status -rule {W_RECON_GROUPS} -status {Waived}         \
+   -expression {(ControlSignal=~"*u_status_23_to_1_sync.*") &&       \
+                (ReconSignal=~"*u_spid_status.outclk_p2s_byte_*")} \
+   -comment {status bits are converged into SPI line. But transmit bit-by-bit. OK to send old and new status bits in a transaction}
+
+set_rule_status -rule {W_CNTL} -status {Waived}   \
```
Comment (tjaychen) [inline hw/top_earlgrey/cdc/cdc_waivers.spi_device.tcl:16]: ah this is the part we talked about if there's a specific naming inside we can waive in general? 
Is there not a rule somewhere in verix where we can inform the tool what is a legal synchronizer?

Comment (eunchan) [reply hw/top_earlgrey/cdc/cdc_waivers.spi_device.tcl:16]: unfortunately, looks like no. But I should read the tool manual thoroughly. Maybe I missed a few commands.

Comment (eunchan) [reply hw/top_earlgrey/cdc/cdc_waivers.spi_device.tcl:16]: `set_user_cntl_synchronizer` may be this is what you expect. I will take a look

**Review by tjaychen — APPROVED** [review]

File: hw/top_earlgrey/cdc/cdc_waivers.spi_device.tcl (hunk 1 above)
Comment (tjaychen) [reply hw/top_earlgrey/cdc/cdc_waivers.spi_device.tcl:16]: yeah but we can merge this first.

Comment (msfschaffner) [reply hw/top_earlgrey/cdc/cdc_waivers.spi_device.tcl:16]: Looks like this is addressed in #11655? 
Does it make sense to merge the common waivers first and then land this without the `W_CNTL` waivers for paths through the synchronizer?

Comment (eunchan) [reply hw/top_earlgrey/cdc/cdc_waivers.spi_device.tcl:16]: The PR #11655 does not work at this time. I am figuring out how nicely I can specify uniquified synchronizer modules.

Comment (msfschaffner) [reply hw/top_earlgrey/cdc/cdc_waivers.spi_device.tcl:16]: Oh I see. Nevermind then.

**Review by msfschaffner — APPROVED** [review]

**Pull request merged**